server/src/index.ts

```diff
@@ -89,7 +89,7 @@ const typeDefs = gql`
   }
 
   type AppLogsResult {
-    logs: String!
+    logs: [String]!
```

Comment:
for better type checking
```suggestion
    logs: [String!]!
```